workspace/workspace.go

```diff
@@ -216,15 +226,68 @@ func LoadWorkspace() (Workspace, error) {
 	manifestFile := filepath.Join(workspacePath, ".yourbase.yml")
 	configFile := filepath.Join(workspacePath, "config.yml")
 
+	var w Workspace
+
 	if PathExists(manifestFile) {
 		log.Debugf("Loading workspace from manifest file: %s", manifestFile)
-		return loadWorkspaceFromPackage(manifestFile, workspacePath)
+		w, err = loadWorkspaceFromPackage(manifestFile, workspacePath)
+	}
+
+	if err == nil && validWorkspaceConfig(w) {
```

Comment:
It seems like the less surprising functionality would be to return failure if a configuration file exists but is invalid. WDYT?

Reply:
That it could get in the way of the developer/user, but I can come up with a way to do that, will add as a TODO